app/models/company.rb

```diff
@@ -5,8 +5,11 @@ class Company < ActiveRecord::Base
   validates :name, :organization, :title, :state, :city, presence: true
   validates :status, inclusion: ["contacted", "uncontacted", "dead", "confirmed"]
   validates :email, presence: true, length: { maximum: 255 },
-                    format: { with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i, }
-                    validates :hire_count, numericality: { only_integer: true, }
+                    format: { with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i, },
+                    uniqueness: true
```

Comment:
Added uniqueness true here so it fails here (and notifies the user) rather than fails in some after_create callback, where the user doesn't know if their account was created or not, and no email comes through....